### ISSUE TITLE
Migrate to vlid.js

### ISSUE THREAD

**Issue by raikasdev** [rittaschool/ritta-server#85]
I recommend switching to [vlid.js](https://github.com/vlucas/vlid) for optimizing purposes. It is a similar library, but 10x smaller and better for as an example the frontend where the 140kb of joi would be packed into our JS file, thy making loading times longer.

**Comment by kymppi** [comment]
Also when i read the description again, i got confused you talking about frontend and packed JS files?. This is the server, right repo?

**Comment by raikasdev** [comment]
> Also when i read the description again, i got confused you talking about frontend and packed JS files?. This is the server, right repo?

Yes, but as it is also included in the shared it will also be in the frontend, and thy will be in the webpacked JS files.
I put it here as this is the "main" repo, and joi is also present here bit more complicatedly and needs more work.

**Comment by raikasdev** [comment]
vlid.js seems abandoned, closing.
